buildpipeline/DotNet-CoreClr-Trusted-FreeBSD.json

```diff
@@ -0,0 +1,324 @@
+      "inputs": {
+        "workItemType": "234347",
+        "assignToRequestor": "true",
+        "additionalFields": "{}"
+      }
+    },
+    {
+      "enabled": false,
+      "definition": {
+        "id": "57578776-4c22-4526-aeb0-86b6da17ee9c"
+      },
+      "inputs": {
+        "additionalFields": "{}"
+      }
+    },
+    {
+      "enabled": false,
+      "definition": {
+        "id": "5d58cc01-7c75-450c-be18-a388ddb129ec"
+      },
+      "inputs": {
+        "branchFilters": "[\"+refs/heads/*\"]",
+        "additionalFields": "{}"
+      }
+    }
+  ],
+  "variables": {
+    "DotNetBootstrapCliTarPath": {
+      "value": "/dotnet-sdk-freebsd-x64.tar",
+      "allowOverride": true
+    },    
+    "system.debug": {
+      "value": "false",
+      "allowOverride": true
+    },
+    "PB_BuildType": {
+      "value": "Release"
+    },
+    "Architecture": {
+      "value": "x64"
+    },
+    "CloudDropAccountName": {
+      "value": "dotnetbuildoutput"
+    },
+    "CloudDropAccessToken": {
+      "value": null,
+      "isSecret": true
+    },
+    "OfficialBuildId": {
+      "value": "$(Build.BuildNumber)"
+    },
+    "Label": {
+      "value": "$(Build.BuildNumber)"
+    }
+  },
+  "demands": [
+    "Agent.OS -equals FreeBSD"
+  ],
+  "retentionRules": [
+    {
+      "branches": [
+        "+refs/heads/*"
+      ],
+      "artifacts": [],
+      "artifactTypesToDelete": [
+        "FilePath",
+        "SymbolStore"
+      ],
+      "daysToKeep": 10,
+      "minimumToKeep": 1,
+      "deleteBuildRecord": true,
+      "deleteTestResults": true
+    }
+  ],
+  "buildNumberFormat": "$(date:yyyyMMdd)$(rev:-rr)",
+  "jobAuthorizationScope": "projectCollection",
+  "jobTimeoutInMinutes": 60,
+  "jobCancelTimeoutInMinutes": 5,
+  "repository": {
+    "properties": {
+      "labelSources": "0",
+      "reportBuildStatus": "true",
+      "fetchDepth": "0",
+      "gitLfsSupport": "false",
+      "skipSyncSource": "false",
+      "cleanOptions": "0",
+      "checkoutNestedSubmodules": "false",
+      "labelSourcesFormat": "$(build.buildNumber)"
+    },
+    "id": "670e3783-ab4f-44fc-9786-d332007da311",
+    "type": "TfsGit",
+    "name": "DotNet-CoreCLR-Trusted",
+    "url": "https://devdiv.visualstudio.com/DevDiv/_git/DotNet-CoreCLR-Trusted",
+    "defaultBranch": "refs/heads/master",
+    "clean": "true",
+    "checkoutSubmodules": false
+  },
+  "processParameters": {},
+  "quality": "definition",
+  "drafts": [],
+  "queue": {
+    "_links": {
+      "self": {
+        "href": "https://devdiv.visualstudio.com/_apis/build/Queues/832"
+      }
+    },
+    "id": 832,
+    "name": "DotNetCore-Infra",
+    "url": "https://devdiv.visualstudio.com/_apis/build/Queues/832",
+    "pool": {
+      "id": 135,
+      "name": "DotNetCore-Infra"
+    }
+  },
+  "id": 1680,
+  "name": "DotNet-CoreClr-Trusted-FreeBSD",
+  "path": "\\",
+  "type": "build",
+  "queueStatus": "enabled",
+  "project": {
+    "id": "0bdbc590-a062-4c3f-b0f6-9383f67865ee",
```

Comment:
There are a lot of GUIDs in this file.  I'm trusting that they're right; not that I know what right means.

Similar with all the numeric ID values.

Reply:
project in this case means "which VSTS it's on".  Many other GUIDS are unique ways to identify tasks.  I've tested this stuff and other than the "Subtype" being wrong (for both this and mac I think)I believe this is all that's needed for official build to work.  One's starting shortly.